gulpfile.js

```diff
@@ -146,7 +147,45 @@ gulp.task('move-data', () => {
 	gulp.src('data/**/*').pipe(gulp.dest(`${DIST_FOLDER}/data`));
 });
 
-gulp.task('default', ['clean'], callback =>
+gulp.task('set-dev-node-env', () => process.env.NODE_ENV = 'development'); // eslint-disable-line no-return-assign
+gulp.task('set-stage-node-env', () => process.env.NODE_ENV = 'development'); // eslint-disable-line no-return-assign
+gulp.task('set-prod-node-env', () => process.env.NODE_ENV = 'production'); // eslint-disable-line no-return-assign
```

Comment:
Is it possible to disable eslint rules for a block instead of for one line at a time?